packages/patternfly-4/README.md

```diff
@@ -0,0 +1,3 @@
+# Packages
+
+Placeholder for description of PatternFly 4 packages
```

Comment:
Can we create dedicated issues for filling this (and the similar PF3 readme) out, just so that effort doesn't get lose in the mix? I realize this PR is geared toward organizing and tweaking existing info, not necessarily giving all the content an overhaul so don't want to hold up getting this PR merged just for that. If we can just capture the issues for now that would ensure we don't forget and leave this placeholder around for longer than we have to.

Reply:
I have opened an issue #635.  That should cover that.